### ISSUE TITLE
Research clustering options

### ISSUE THREAD

**Issue by ktjnyc** [hackforla/food-oasis#869]
### Overview
We need to research the clustering options for food locations so that it's simple and easy for food seekers to find a location.  

### Action Items

- [x] Collect usability information to determine needs/points of friction #892 
- [x] Research best practices of clustering map results and provide updates in the comments
- [x] Jenny can connect Natalie to a potential team that may have info 

### Resources/Instructions
REPLACE THIS TEXT -If there is a website which has documentation that helps with this issue provide the link(s) here.
[Link to article on pins vs clusters](https://medium.com/@letstalkproduct/the-map-search-experience-pins-vs-clusters-b3d18d8159c5
)

**Comment by ktjnyc** [comment]
Natalie to update with notes

**Comment by fancyham** [comment]
One thing to try might be to have clustering only when zoomed way, way out when performance would take a hit trying to show all 800 locations.

**Comment by nkiwan** [comment]
After pretty extensive research, I wasn’t able to find any UX research data on map clustering, though I found plenty of articles on the topic in general. The consensus could be summed up this way: Depending on your use cases and user base, clustering can be a helpful way to declutter a map that has either a lot of location markers, a lot of overlapping markers, or both. 

I think we need to pay particular attention to the specifics of when we use clustering (at which zoom levels, etc), if we decide to use it at all. Clicking on (or touching) a cluster should zoom in to that area and get close enough to show the actual location markers. I like Bryan’s suggestion of only using clustering when the map is zoomed so far out that showing every location would take too much bandwidth. 

My initial thoughts are that we probably don’t have enough location markers for possible clustering benefits to outweigh the confusion it may cause for users. I think the visual problem of a lot of markers close together is something most users would either know how to solve, or be able to figure out without too much difficulty. Most of the examples of use cases I found were for maps that have hundreds, or even thousands, of location markers. 

For folks who want to do more reading on this, these were the most useful resources I found:
https://medium.com/@alessa.gross/too-many-markers-revisited-e9d99f575fea
https://www.jameslmilner.com/post/dense-spatial-data/
This research paper gets pretty technical: https://www.researchgate.net/publication/334472260_Rethinking_the_usage_and_experience_of_clustering_markers_in_web_mapping
